src/views/virtualmachines/tree/hooks/useTreeViewData.ts

```diff
@@ -65,11 +78,12 @@ export const useTreeViewData = (): UseTreeViewData => {
     namespaced: true,
   });
 
+  // if multicluster, we should have access to all VMIM resources
   const allowedVMIMResources = useK8sWatchResources<{
     [key: string]: V1VirtualMachineInstanceMigration[];
   }>(
     Object.fromEntries(
-      projectNamesLoaded && !isAdmin
+      projectNamesLoaded && !isAdmin && !supportsMulticluster
```

Comment:
Q: this assumes that this filtering won't be necessary whenever supportsMulticluster is active. That a fair assumption?

Reply:
Yup, I agree that is true.